cves/kernel/CVE-2012-6544.yml

```diff
@@ -55,7 +55,11 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
```

Comment:
Make sure the description does not include any project specific information. I think there is too much jargon here.

Reply:
I agree, I think the description should be general and simply describe the vulnerability; for example: "In the Linux kernel, the bluetooth subsystem had an issue where the memory was not properly being cleared after use, leading to a leakage of sensitive information to the user".